rpc-jobs/RPC-AIO.yml

```diff
@@ -87,7 +87,7 @@
     concurrent: true
     properties:
       - build-discarder:
-          num-to-keep: 30
+          num-to-keep: 7
```

Comment:
This should actually be `days-to-keep` and not `num-to-keep` if we want to do it by days. Not sure if that's what it was originally supposed to be or not.